### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [pretalx/pretalx#1181] [General]
Would a Mastodon account for the project be possible?

**Comment by rixx** [comment]
What purpose would that account serve?

**Comment by linux-lukas** [comment]
Mainly probably project information and project announcements (Ex: "We are looking for testers for a function!", "We want to do a load test - who has time and desire?"). Secondary for interaction with the pretalx community. @rixx

**Reply by rixx** [reply]
I don't think a separate account is feasible at this time. Since I'm the primary maintainer and nearly sole developer at this point, it's easier when people get in touch with me directly if they want to contact anybody on fedi, and it wouldn't force me to put effort into another account to regularly check and update.

I'm very willing to revisit this when and if the pretalx development team expands.
